From matthew@wil.cx  Fri Oct  9 10:52:24 2009
From: Matthew Wilcox <matthew@wil.cx>
Date: Thu, 24 Sep 2009 16:18:27 -0600
Subject: USB: Convert a dev_info to a dev_dbg
To: Alan Stern <stern@rowland.harvard.edu>
Cc: linux-scsi@vger.kernel.org, linux-usb@vger.kernel.org
Message-ID: <20090924221827.GB7943@parisc-linux.org>
Content-Disposition: inline


From: Matthew Wilcox <willy@linux.intel.com>

Knowing which configuration was chosen is a debugging aid more than it
is informational.

Signed-off-by: Matthew Wilcox <willy@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/generic.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/core/generic.c
+++ b/drivers/usb/core/generic.c
@@ -139,7 +139,7 @@ int usb_choose_configuration(struct usb_
 
 	if (best) {
 		i = best->desc.bConfigurationValue;
-		dev_info(&udev->dev,
+		dev_dbg(&udev->dev,
 			"configuration #%d chosen from %d choice%s\n",
 			i, num_configs, plural(num_configs));
 	} else {
